spec/index.html

```diff
@@ -5688,30 +5688,45 @@ <h5>COALESCE</h5>
             <h5>NOT EXISTS and EXISTS</h5>
             <p>There is a filter operator <code>EXISTS</code> that takes a graph pattern.
               <code>EXISTS</code> returns <code>true</code>/<code>false</code> depending on whether the
-              pattern matches the dataset given the bindings in the current group graph pattern, the
-              dataset and the <a href="#defn_ActiveGraph">active graph</a> at this point in the query
-              evaluation. No additional binding of variables occurs. The <code>NOT EXISTS</code> form
-              translates into <code>fn:not(EXISTS{...})</code>.</p>
+              pattern matches the dataset given solution mapping with respect to which
+              the expression is <a href="#expression-evaluation">evaluated</a>.
```

Comment:
```suggestion
```

Github is not letting me write a suggestion to changes two lines.
The suggestion above is the rewording of both lines.
This suggestion deletes the original second line.

Reply:
Yeah, the "new PR experience" has lots of problems, including inability to change 2 lines with 1 comment.

I've been avoiding the new as much as possible. Which often means I'm clicking back and forth between the old and new, because *both* tell me there are comments I cannot see in the one and I should check the other.